app/src/main/kotlin/com/wire/android/ui/common/Logo.kt

```diff
@@ -31,7 +31,7 @@ import com.wire.android.R
 fun Logo(modifier: Modifier = Modifier) {
     Image(
         painter = painterResource(id = R.drawable.ic_wire_logo),
-        contentDescription = stringResource(id = R.string.content_description_app_logo),
+        contentDescription = null,
```

Comment:
Can we remove the string from strings file ?

Reply:
yeah, It's not used anymore, so bye :)